src/db/models/Transaction.ts

```diff
@@ -46,34 +33,26 @@ Transaction.init({
     type: DataTypes.DATE,
     allowNull: true
   },
-  categoryID: {
-    type: DataTypes.INTEGER.UNSIGNED,
-    allowNull: true,
-  },
-  payeeID: {
-    type: DataTypes.INTEGER.UNSIGNED,
-    allowNull: true,
-  },
 }, {
   sequelize,
   tableName: 'transactions',
 });
 
 
-Transaction.hasOne(Account, {
-  sourceKey: 'id',
-  foreignKey: 'accountID',
-  as: 'Account',
-});
-
-Transaction.hasOne(Account, {
-  sourceKey: 'id',
-  foreignKey: 'toAccountID',
-  as: 'ToAccount',
-});
-
-Transaction.hasOne(Category,{
-  sourceKey: 'id',
-  foreignKey: 'categoryID',
-  as: 'Category',
-});
+// Transaction.hasOne(Account, {
+//   sourceKey: 'id',
+//   foreignKey: 'accountID',
+//   as: 'Account',
+// });
+//
+// Transaction.hasOne(Account, {
+//   sourceKey: 'id',
+//   foreignKey: 'toAccountID',
+//   as: 'ToAccount',
+// });
+//
+// Transaction.hasOne(Category,{
+//   sourceKey: 'id',
+//   foreignKey: 'categoryID',
+//   as: 'Category',
+// });
```

Comment:
I do not think changing from hasOne to hasMany will get benifits, 

but anyway it is fine if it works well.


merge to master, then to prod to deploy